src/ansys/additive/core/machine.py

```diff
@@ -105,6 +125,8 @@ def __init__(
         layer_rotation_angle: float = MachineConstants.DEFAULT_LAYER_ROTATION_ANGLE,
         hatch_spacing: float = MachineConstants.DEFAULT_HATCH_SPACING,
         slicing_stripe_width: float = MachineConstants.DEFAULT_SLICING_STRIPE_WIDTH,
+        heat_source_model: str = MachineConstants.DEFAULT_HEAT_SOURCE_MODEL_NAME,
+        ring_mode_coefficient_set_index: int = MachineConstants.DEFAULT_RING_COEFFICIENTS_SET_INDEX,
```

Comment:
How do pyadditive users know what the different coefficient set values are? We should probably have them documented somewhere and have a link to that documentation in the doc string for this class.